templates/base/packages/nextjs/public/manifest.json.template.mjs

```diff
@@ -0,0 +1,15 @@
+import { stringify, withDefaults } from '../../../../utils.js'
+
+const contents = ({ name, description, iconPath, extraContent }) => stringify({
+  name: name[0],
+  description: description[0],
+  ...(iconPath[0] && { iconPath: iconPath[0] }), // Only include iconPath if it's provided
```

Comment:
If extension developer doesn't pass the iconPath it will default to "logo.svg" right? So we don't need this check?

Reply:
I think instead of icon, we are gonna have icons, like the example below:
https://github.com/serwist/serwist/blob/main/examples/next-web-push/public/manifest.json
So, we don't need the icon if the dev does not provide it.

Reply:
Ohh yeah makes sense! So people need to pas `export const iconPath = ''` to omit that field. 

What if people forgot to add ``export const iconPath = ''`` but also have `icons` in their template, will it still work as expected?

Reply:
https://github.com/portdeveloper/test-everything

yes i tested it with this extension which has all the args mjs files for the changed files in this PR